Sming/SmingCore/Network/TcpConnection.cpp

```diff
@@ -300,8 +300,14 @@ void TcpConnection::initialize(tcp_pcb* pcb)
 	tcp = pcb;
 	sleep = 0;
 	canSend = true;
+	
 #ifdef ENABLE_SSL
-	axl_init(10);
+	static bool axl_initialized = false;
```

Comment:
@Informatic Nice catch. It will be better to add reference counting here and use this reference counting in the destructor (if refcount == 0 then axl_free  && axl_initialized = false). Can you please, make these changes and submit it as part of this PR?